contracts/hts-precompile/examples/token-transfer/TokenTransferContract.sol

```diff
@@ -70,4 +70,22 @@ contract TokenTransferContract is FeeHelper {
             revert ();
         }
     }
+
+    function associateTokenPublic(address account, address token) public returns (int responseCode) {
```

Comment:
These are flipped.
These functions already exist under create and the ones you've added under create should be under transfer.

In terms of organization what we're going for is
- `TokenCreate` - operations needs to create a token and first enable accounts to transact (associate, kyc, unfreeze)
- `TokenTransfer` - operations to transfer assets or directly modify transfer operations (approve)
- `TokenManagement` - operations than modify a token characteristics or modify an accounts ability to transfer after setup
- `TokenQuery` - all query operations 

That's the suggestion to modularize the many HTS operations but also keep the contracts manageable in size.
Will look to add this to the README

Reply:
Most functions can not be done without `delegatecall`, and we must add them.